code/stable/gooltest/swift/HelloWorld/main.swift

```diff
@@ -120,6 +120,7 @@ else if b == 5 {
     c -= 1
     b -= 1
     var myList: [Int] = []
+    var myObj: Character = "o"
```

Comment:
Is this intentional or a side effect of something else? Not saying it's wrong, just curious

Reply:
That's there because before Julia, `HelloWorld.hs` was producing that line, using `objDecDef` or something like that.  When I added Julia, which does not implement `objDecDef`, I commented out the line since it's the only line in the program that isn't procedural-friendly.  Now that I've moved Julia over, there need to be two versions of `HelloWorld.hs` - one for OO languages and one for procedural ones.  Because of that, I removed the commented-out line in the procedural version , and uncommented it in the OO version.